python/ray/tune/tuner.py

```diff
@@ -28,6 +28,14 @@
 _SELF = "self"
 
 
+_TUNER_FAILED_MSG = (
```

Comment:
hmm is it possible to surface the error here?

Reply:
We do a `raise TuneError(_TUNER_FAILED_MSG) from original_exception` so the error is directly available. 

The reason why I'd like to not repeat the full error message here is that it could be a full RayTaskError traceback, so very long, which will clutter the output